homeassistant/components/matter/light.py

```diff
@@ -421,6 +421,7 @@ class MatterLightEntityDescription(
         subscribe_attributes=(
             clusters.OnOff.Attributes.OnOff,
             clusters.LevelControl.Attributes.CurrentLevel,
+            clusters.ColorControl.Attributes.ColorMode,
```

Comment:
Is this a new feature or a fix?

Reply:
That is a fix for color temperature lights not being detected properly, probably should have done that in a different pull request

Reply:
Ok. Is this case covered now with the new tests in this PR?

Reply:
It is, as I only figured out it was an issue when I loaded the color temperature light dumps

Reply:
Ok, then I think we can keep it in this PR, as the fix is now tested.